mlflow/utils/process.py

```diff
@@ -44,8 +45,13 @@ def _exec_cmd(
                       If this argument is specified, `kwargs` cannot contain `env`.
     :param: capture_output: If True, stdout and stderr will be captured and included in an exception
                             message on failure; if False, these streams won't be captured.
-    :param kwargs: Keyword arguments (except `check` and `text`) passed to `subprocess.run`.
-    :return: A `subprocess.CompletedProcess` instance.
+    :param: synchronous: If True, wait process complete and return a CompletedProcess instance,
+                         If False, does not wait process complete and return a Popen instance,
+                         and ignore the `throw_on_error` argument.
+    :param kwargs: Keyword arguments (except `check` and `text`) passed to `subprocess.run` or
+                   `subproces.Popen`.
```

Comment:
```suggestion
    :param kwargs: Keyword arguments (except `text`) passed to 
 `subproces.Popen`.
```

Let's remove `subprocess.run` and `check`.